saga/sagaState_prop_test.go

```diff
@@ -0,0 +1,215 @@
+package saga
+
+import (
+	"bytes"
+	"github.com/leanovate/gopter"
+	"github.com/leanovate/gopter/gen"
+	"github.com/leanovate/gopter/prop"
+	"testing"
+)
+
+func Test_ValidSagaId(t *testing.T) {
```

Comment:
Aren't you basically reimplementing validateSagaId() and validateTaskID() in these tests? It seems like these would be better suited for unit tests if the implementation can't be broken down into different properties. (this is assuming we'll eventually do more than checking for empty string)